cli.js

```diff
@@ -123,10 +119,10 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
   .wrap(null);
 
 const commands = yargs.getCommandInstance().getCommands();
-const argv = yargs.argv;
+// first command pushed to command stack(from user input)
```

Comment:
What strange wizardry is this?  It looks like you're using `getCommandInstance` (which I couldn't find in the API docs), in order to manually perform behaviour I would expect to get from `yargs`.

Reply:
Yea that wizardry is basically what `commander` feels like all the time. I'm not sure why it's not directly supported.

Reply:
"Welcome to the future," basically :-(.